examples/get-started/create-react-app/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-map-gl": "^5.2.1",
+    "react-scripts": "3.3.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
```

Comment:
This is breaking CI. You need to either remove this or add `eslint-config-react-app` to the root `devDependencies`.

Reply:
Thanks. I added the dev dependency.

Reply:
Thanks. I added eslint-config-react-app.

Reply:
You need to add it to the root package.json. CI does not install dependencies for each example.